packages/moonstone/MoonstoneDecorator/MoonstoneDecorator.module.less

```diff
@@ -13,9 +13,11 @@
 @import "../styles/skin.less";
 @import "../styles/variables.less";
 
-/* Prevent browser's default focus treatment (at least in Chrome) */
 :focus {
+	// Prevent Chrome's default focus treatment
 	outline: none;
+	// Prevent Android Chrome from highlighting tapped elements
+	-webkit-tap-highlight-color: transparent;
```

Comment:
Should we apply this into the root to prevent this action on all elements?

Reply:
We should apply this rule and the `:focus` rule to `.root` to keep the rules together and to ensure they only affect our components.